openmetadata-spec/src/main/resources/json/schema/entity/services/connections/dashboard/tableauConnection.json

```diff
@@ -27,16 +65,18 @@
       "type": "string",
       "format": "uri"
     },
-    "username": {
-      "title": "Username",
-      "description": "Username for Tableau.",
-      "type": "string"
-    },
-    "password": {
-      "title": "Password",
-      "description": "Password for Tableau.",
-      "type": "string",
-      "format": "password"
+    "authType": {
+      "mask": true,
```

Comment:
like, when we are specifying the `password` field

Reply:
It is fine. The mask property to true value takes care of if the serialization must take it into account or not the field. We use it when processing events to avoid sending sensitive data.

What we should check is if this is properly stored in DB or not with the password masked, otherwise, update this class: `openmetadata-service/src/main/java/org/openmetadata/service/secrets/converter/ClassConverterFactory.java`.

Reply:
thanks, updating the converter